docs/URLSchemeConfiguring.md

```diff
@@ -58,6 +58,17 @@ private fun handleLinkIntent(intent: Intent?) {
 
 ## Verifying the Custom URL Scheme
 
-Test that the URL scheme handling is set up correctly by navigating to `appcues-APPCUES_APPLICATION_ID://debugger` in your browser on the device with the app installed.
+The Appcues debugger allows you to easily validate that the Appcues deeplink is properly configured.
+
+1. Launch the debugger in your app with a call to ``Appcues/debug()``.
+2. Expand the debugger by tapping the floating button.
+3. Tap the "Appcues Deeplink Configured" row to verify the status. If a checkmark appears, the Appcues deeplink is properly configured.
+
+### Troubleshooting
+
+- `Error 0`: The Appcues debugger was unable to set up the verification test.
+- `Error 1`: The `CFBundleURLSchemes` value is missing. Refer to the "Register the Custom URL Scheme" section above.
```

Comment:
this is referring to the iOS `CFBundleURLSchemes` - should update to reflect the AndroidManifest.xml intent-filter implementation here

Reply:
I literally copied and pasted, my bad